docs/cdn_commands.md

```diff
@@ -61,7 +61,8 @@ if(awsCreds) awsRum.setAwsCredentials(credentialProvider);
 | recordError | Error \|&nbsp;ErrorEvent \|&nbsp;String | `try {...} catch(e) { cwr('recordError', e); }`<br/><br/>`try {...} catch(e) { awsRum.recordError(e); }` | Record a caught error.
 | registerDomEvents | Array | `cwr('registerDomEvents', [{ event: 'click', cssLocator: '[label="label1"]' }]);`<br/><br/>`awsRum.registerDomEvent([{ event: 'click', cssLocator: '[label="label1"]' }]);` | Register target DOM events to record. The target DOM events will be added to existing target DOM events. The parameter type is equivalent to the `events` property type of the [interaction telemetry configuration](https://github.com/aws-observability/aws-rum-web/blob/main/docs/cdn_installation.md#interaction).
 | setAwsCredentials | [Credentials](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html) \|&nbsp;[CredentialProvider](https://www.npmjs.com/package/@aws-sdk/credential-providers) | `cwr('setAwsCredentials', cred);`<br/><br/>`awsRum.setAwsCredentials(cred);` | Forward AWS credentials to the web client. The web client requires AWS credentials with permission to call the `PutRumEvents` API. If you have not set `identityPoolId` and `guestRoleArn` in the web client configuration, you must forward AWS credentials to the web client using this command.
-
+| recordEvent | Object | `cwr('recordEvent', {event_type: 'your_event_type', event_data: {field1: 1, field2: 2}})` <br/><br/> `awsRum.recordEvent('your_event_type', {field1 1, field2: 2})` | Record an event. Learn more in [Using CloudWatch RUM Web Client to Record Custom Events](custom_events.md)
+| addPlugin | Plugin | `cwr('addPlugin', this.myCustomPlugin)` <br/><br/> `awsRum.addPlugin(this.myCustomPlugin)` | Adds a plugin to the Web Client. Learn more in [Using CloudWatch RUM Web Client to Record Custom Events](custom_events.md)
```

Comment:
I'm probably contradicting something I said earlier, but it might be better to do docs in a separate commit. This way we can separate the technical conversation from the doc revisions.